Library/Formula/node.rb

```diff
@@ -21,7 +22,7 @@ class Node < Formula
 
   depends_on :python => :build if MacOS.version <= :snow_leopard
   depends_on "pkg-config" => :build
-  depends_on "openssl" => :optional
```

Comment:
Why the removal of this option? My understanding is that node (by default) statically links its own version of OpenSSL, complete with a compiled-in CA list.